CMakeLists.txt

```diff
@@ -1,31 +1,34 @@
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.14...3.23)
 
 project(magic_enum VERSION "0.8.0" LANGUAGES CXX)
 
+set(CMAKE_CXX_STANDARD 17)
+
 if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     set(IS_TOPLEVEL_PROJECT TRUE)
 else()
     set(IS_TOPLEVEL_PROJECT FALSE)
 endif()
 
-option(MAGIC_ENUM_OPT_ENABLE_NONASCII "Enable support for non-ASCII enumeration identifier" ${IS_TOPLEVEL_PROJECT})
+option(MAGIC_ENUM_OPT_ENABLE_NONASCII "Enable support for non-ASCII enumeration identifier" YES)
 option(MAGIC_ENUM_OPT_BUILD_EXAMPLES "Build magic_enum examples" ${IS_TOPLEVEL_PROJECT})
 option(MAGIC_ENUM_OPT_BUILD_TESTS "Build and perform magic_enum tests" ${IS_TOPLEVEL_PROJECT})
 option(MAGIC_ENUM_OPT_INSTALL "Generate and install magic_enum target" ${IS_TOPLEVEL_PROJECT})
 
+enable_testing()
```

Comment:
Why is testing here? And not where is was used, like before?

Reply:
the example is use to test  installed version using ctest